### PR TITLE
Add X32 edit version 3.2

### PR DESCRIPTION
**Pull request by marcoslater**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-drivers/pulls
[closed issues]: https://github.com/caskroom/homebrew-drivers/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/x32-edit.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'x32-edit' do
+  version '3.2'
+  sha256 '2dee264d0df054e4f9b689af99046b7b28f5b8519cf1fba8df103ca30edcdc0a'
+
+  # Taken from http://www.music-group.com/brand/c/Behringer/downloads?active=Downloads
```
Comment (commitay) [inline Casks/x32-edit.rb]: Remove this line.

**Comment by commitay** [comment]
If this is ever updated will it be X33-Edit (version 3.3) or is X32 the actual name?

**Comment by marcoslater** [comment]
@commitay X32 is the mixer product name. "32" stands for 32 inputs. 

They have other lines of mixers with different inputs, but they have different software to control them, some which is not available for mac unfortunately. 

The version number being the same is a happy coincidence.

File: Casks/x32-edit.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'x32-edit' do
+  version '3.2'
```
Comment (commitay) [inline Casks/x32-edit.rb]: Please also add a comment above this line.

```
cask 'behringer-x32-edit' do
  # note: "32" is not a version number, but an intrinsic part of the product name
  version '3.2'
```

**Pull request merged**